src/js/loading-spinner.js

```diff
@@ -17,10 +18,20 @@ class LoadingSpinner extends Component {
    *         The dom element that gets created.
    */
   createEl() {
+    const isAudio = this.player_.isAudio();
+    const playerType = this.localize(isAudio ? 'Audio Player' : 'Video Player');
+
     return super.createEl('div', {
       className: 'vjs-loading-spinner',
       dir: 'ltr'
-    });
+    }, {
+      role: 'alert'
```

Comment:
I'm not sure we want this, at least not yet.

Reply:
Yeah, me either. I wanted to see how it behaved.